event-subscription/src/main/java/uk/gov/justice/subscription/SubscriptionHelper.java

```diff
@@ -25,7 +25,7 @@ private Integer getPrioritisationNumber(final String prioritisation) {
             }
             return HIGHEST_PRIORITY;
         } catch (final NumberFormatException nfe) {
-            throw new YamlParserException(format("Incorrect prioritisation number: %s in subscription-descriptor.yaml", prioritisation));
+            throw new YamlParserException(format("Incorrect prioritisation number: %s in subscription-descriptor.yaml", prioritisation),nfe);
```

Comment:
minor...but there could be a space after the comma

Reply:
done